lib/ui/widgets/mini_player_widget.dart

```diff
@@ -62,7 +62,7 @@ class _MiniPlayerBuilderState extends State<_MiniPlayerBuilder> with SingleTicke
 
   @override
   Widget build(BuildContext context) {
```

Comment:
Hi @amugofjava,

I see that these changes to mini_player_widget are not reflected to master.
Using accentTextTheme here and setting play_pause icon color to iconTheme.color